src/v/cloud_storage/remote_segment_index.cc

```diff
@@ -279,7 +290,11 @@ iobuf offset_index::to_iobuf() {
       .rp_index = _rp_index.copy(),
       .kaf_index = _kaf_index.copy(),
       .file_index = _file_index.copy(),
-    };
+      .base_time = _initial_time.value(),
+      .last_time = _time_index.get_last_value(),
+      .time_write_buf = std::vector<int64_t>(
+        _file_offsets.begin(), _file_offsets.end()),
```

Comment:
didn't fully read the code, but is this like right, from _file_offsets to time_write_buf?

Reply:
Oops, copy paste error.  Fixed.